docs/shared/recipes/root-level-scripts.md

```diff
@@ -0,0 +1,71 @@
+# Run Root-Level NPM Scripts with Nx
+
+There are often tasks in a codebase that apply to the whole codebase rather than a single project. Before version 15.3.0, the only way to use Nx's task pipelines or caching mechanisms for those scripts was to create an empty `scripts` project and run the scripts from there. Now you can run those commands directly from the root `package.json`.
+
+## Example
+
+Let's say your root `package.json` looks like this:
+
+```json
+{
+  "name": "myorg",
+  "scripts": {
+    "echo": "echo hello world"
```

Comment:
I'd use a target that creates some files so we can show how to configure inputs and ouputs